### PR TITLE
Actively monitor related bugs

### PR DESCRIPTION
**Pull request by wuhuiluo**
## What this PR does / why we need it:
1. Remove the background color of the Add button for exception checking 
2. Edit active monitoring, and the last saved value is displayed in the header 
3. In advanced settings, the number of retransmissions cannot be limited to decimal    
4. Edit the Title Modification of the active monitoring page 
5. Repair the monitoring details page and click Edit. The exception check in advanced settings cannot be edited

## Does this PR introduce a user interface change?

❎ No

## ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |    1. Remove the background color of the Add button for exception checking 2. Edit active monitoring, and the last saved value is displayed in the header 3. In advanced settings, the number of retransmissions cannot be limited to decimal    4.  Edit the Title Modification of the active monitoring page 5.  Repair the monitoring details page and click Edit. The exception check in advanced settings cannot be edited   |
| 🇨🇳 中文    |      1. 去掉异常检查下方添加按钮的底色 2. 编辑主动监控，header中显示上次保存的值 3. 高级设置中，重发次数限制不能输入小数  4. 编辑主动监控页面的标题修改 5. 修复监控详情页面，点击编辑，高级设置中异常检查无法编辑 |

## Does this PR need be patched to older version?
✅ Yes(version is required)
relase/1.4

## Which issue(s) this PR fixes:

Fixes #

### CODE REVIEW

File: shell/app/locales/zh.json [hunk 1]
```diff
@@ -2128,6 +2128,7 @@
     "earlier": "更早",
     "edit a endpoint": "编辑流量入口",
     "emergency": "紧急",
+    "edit monitoring": "编辑监控",
```
Comment (McDaddy) [inline shell/app/locales/zh.json:2131]: where did u refer this translation?

File: shell/app/modules/msp/monitor/status-insight/pages/status/add-modal.tsx [hunk 2]
```diff
@@ -484,9 +485,13 @@ const AddModal = (props: IProps) => {
...
                           onChange={(v) => {
                             if (v === 'http_code') {
-                              condition[index].operate = '>=';
+                              const newCondition = [...condition];
+                              newCondition[index] = { ...newCondition[index], operate: '>=' };
+                              updater.condition(newCondition);
                             } else {
-                              condition[index].operate = 'contains';
+                              const newCondition = [...condition];
+                              newCondition[index] = { ...newCondition[index], operate: 'contains' };
```
Comment (McDaddy) [inline shell/app/modules/msp/monitor/status-insight/pages/status/add-modal.tsx:493]: use `immer` to edit the nested state

File: shell/app/locales/zh.json (hunk 1 above)
Comment (wuhuiluo) [reply shell/app/locales/zh.json:2131]: The code has just been updated. The conflict resolution side has overwritten the previous modifications

File: shell/app/modules/msp/monitor/status-insight/pages/status/add-modal.tsx (hunk 2 above)
Comment (wuhuiluo) [reply shell/app/modules/msp/monitor/status-insight/pages/status/add-modal.tsx]: ok,modified

File: shell/app/modules/msp/monitor/status-insight/pages/status/add-modal.tsx [hunk 3]
```diff
@@ -341,7 +353,12 @@ const AddModal = (props: IProps) => {
                   updater.headers({ ...headers, 'Content-Type': textType });
                 }
                 if (key === '2' && bodyType === noneType) {
-                  updater.headers({});
+                  if (headers['Content-Type'] === ('x-www-form-urlencoded' || 'text/plain' || 'application/json')) {
+                    updater.headers({});
+                  } else {
+                    // eslint-disable-next-line no-useless-return
+                    return;
```
Comment (McDaddy) [inline shell/app/modules/msp/monitor/status-insight/pages/status/add-modal.tsx:360]: why keep this warning

File: shell/app/modules/msp/monitor/status-insight/pages/status/status.tsx [hunk 4]
```diff
@@ -37,10 +37,11 @@ const Status = () => {
 
   const { type = 'All' } = query || {};
 
-  const [{ modalVisible, formData, filterType }, updater, update] = useUpdate({
+  const [{ modalVisible, formData, filterType, mode }, updater, update] = useUpdate({
     modalVisible: false,
     formData: null as MONITOR_STATUS.IMetricsBody | null,
     filterType: type,
+    mode: '',
```
Comment (McDaddy) [inline shell/app/modules/msp/monitor/status-insight/pages/status/status.tsx:44]: we check whether its add/edit by whether formData is empty. no need add extra state

File: shell/app/modules/msp/monitor/status-insight/pages/status/add-modal.tsx [hunk 5]
```diff
@@ -545,11 +568,18 @@ const AddModal = (props: IProps) => {
...
               </div>
 
-              <Button className="mt-4" size="small" type="primary" onClick={addItem}>
+              <Button ghost className="mt-4" size="small" type="primary" onClick={addItem}>
                 {i18n.t('common:add')}
               </Button>
               <h4 className="mt-4 mb-3 text-sm">{i18n.t('msp:number of retries')}</h4>
-              <InputNumber value={retry} min={0} max={10} defaultValue={2} onChange={(v) => updater.retry(v)} />
+              <InputNumber
+                parser={(formatRetry: string) => (/^\d+$/.test(formatRetry) ? formatRetry : formatRetry[0])}
```
Comment (McDaddy) [inline shell/app/modules/msp/monitor/status-insight/pages/status/add-modal.tsx:576]: what does this mean? keep no float?  try `precision ` instead

File: shell/app/modules/msp/monitor/status-insight/pages/status/add-modal.tsx (hunk 3 above)
Comment (wuhuiluo) [reply shell/app/modules/msp/monitor/status-insight/pages/status/add-modal.tsx]: modified

File: shell/app/modules/msp/monitor/status-insight/pages/status/status.tsx (hunk 4 above)
Comment (wuhuiluo) [reply shell/app/modules/msp/monitor/status-insight/pages/status/status.tsx:44]: ok,modified

File: shell/app/modules/msp/monitor/status-insight/pages/status/add-modal.tsx (hunk 5 above)
Comment (wuhuiluo) [reply shell/app/modules/msp/monitor/status-insight/pages/status/add-modal.tsx]: ok,modified

File: shell/app/modules/msp/monitor/status-insight/pages/status/status.tsx (hunk 4 above)
Comment (McDaddy) [reply shell/app/modules/msp/monitor/status-insight/pages/status/status.tsx:44]: modified but state still there?

Comment (wuhuiluo) [reply shell/app/modules/msp/monitor/status-insight/pages/status/status.tsx]: mine,now modified

**Pull request merged**